src/blocks/carousel/view.php

```diff
@@ -259,8 +249,9 @@ function newspack_blocks_render_block_carousel( $attributes ) {
 		);
 	}
 
-	$slides_per_view = absint( ! empty( $attributes['slidesPerView'] ) ? $attributes['slidesPerView'] : 1 );
+	$slides_per_view = absint( $attributes['slidesPerView'] ) ?? 1;
 	$slides_to_show  = $slides_per_view <= $counter ? $slides_per_view : $counter;
+	$aspect_ratio    = floatval( $attributes['aspectRatio'] ) ?? 0.75;
```

Comment:
The `??` operator is like `isset()`, so it'll never reach `1` or `0.75` after `absint()` or `floatval()`. They will be stored as `0` in both cases and cause division by zero error on line 266.

To check if the attribute is set and have a default, this should work:

```suggestion
	$slides_per_view = absint( $attributes['slidesPerView'] ?? 1 );
	$slides_to_show  = $slides_per_view <= $counter ? $slides_per_view : $counter;
	$aspect_ratio    = floatval( $attributes['aspectRatio'] ?? 0.75 ) ;
```